Dockerfile

```
@@ -1,28 +1,17 @@
+COPY . /app/
 
 # Install Node.js dependencies
 RUN npm install
 
-# Copy frontend source
-COPY src/ /app/src/
-
-COPY --from=ghcr.io/astral-sh/uv:latest /uv /usr/local/bin/uv
-
 # Build the frontend
 RUN npm run build
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)